services/security/providerTypes.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache v2.0 License.
+
+package security
+
+import (
+	"github.com/microsoft/moc/pkg/errors"
+	pbcom "github.com/microsoft/moc/rpc/common"
+)
+
+func GetProviderType(pbProvider pbcom.ProviderType) ProviderType {
+	providerInt := int32(pbProvider)
+	value, found := pbcom.ProviderType_name[providerInt]
+	if !found {
+		return AnyProviderType // Not found, return AnyProvider
+	}
+	return ProviderType(value)
+}
+
+func GetWssdProviderType(providerType ProviderType) (pbcom.ProviderType, error) {
```

Comment:
Try to use Moc instead of Wssd in new code.

Reply:
Changed wssd references to moc.